api/organisations/permissions/models.py

```diff
@@ -8,14 +8,38 @@
 
 
 class UserOrganisationPermission(AbstractBasePermissionModel):
-    organisation = models.ForeignKey(Organisation, on_delete=models.CASCADE)
+    organisation = models.ForeignKey(
+        Organisation,
+        on_delete=models.CASCADE,
+        related_query_name="userpermission",
```

Comment:
It's to allow filter on the field in the query set. It is used here: https://github.com/Flagsmith/flagsmith/pull/2232/files#diff-7904b170017aa6b6a88dae1dc0905d9308c5b7b9475020895a44427433d7a60cR127

Reply:
Yep, I'm just asking because I'm not sure why we'd use `related_query_name` explicitly instead of just `related_name`. I guess right now we have no need to do `organisation.userpermission` but it would be consistent with the rest of our ForeignKey implementations. 

Also, should it be `userpermissions` rather than `userpermission`?

Reply:
I think they kind of have different use cases, e.g: for permissions we use related_query_name because we need to filter on those fields, e.g: https://github.com/Flagsmith/flagsmith/blob/main/api/projects/models.py#L149 

> Also, should it be userpermissions rather than userpermission?

No, they are supposed to be singular, right? Because in the query they point to one row.
e.g: https://docs.djangoproject.com/en/4.2/ref/models/fields/#django.db.models.ForeignKey.related_query_name